### DISCUSSION TITLE
My ambilight box

### DISCUSSION THREAD

**Discussion by AmbiMod** [awawa-dev/HyperHDR#80] [Show and tell]
Hi awawa.

I’m working on an ambilight box and i would like to share some pictures with you.

it’s almost finished, just few things to adjust and i can start a full print

![281C1925-D46F-4B54-A8A8-0740A4BC20C2](https://user-images.githubusercontent.com/26428901/124399767-f8a0a600-dd1d-11eb-8d6c-8f5609d97a1a.jpeg)
![BEBB8A5D-B8FE-443C-AA41-76BE29C750C8](https://user-images.githubusercontent.com/26428901/124399769-f9d1d300-dd1d-11eb-9a15-ca46b5455a63.jpeg)
![F862368D-60C3-4A49-AED3-F6BE53155ABF](https://user-images.githubusercontent.com/26428901/124399770-f9d1d300-dd1d-11eb-9653-f7ea4c8a5576.jpeg)
![77DF1B4D-5FA1-496E-AB2A-651BBCAC3B77](https://user-images.githubusercontent.com/26428901/124399771-fa6a6980-dd1d-11eb-99ac-bddc3c5d0d9c.jpeg)

I hope you like it.

**Comment by awawa-dev** [comment]
Nice! The power supply is external or do you have a plan for a version with IEC 320 C14 fuse power socket?

**Reply by AmbiMod** [reply]
In fact i will use two things : 
- Two GX12 connector, one 2 pin for the power and another one with 4 or 6 pins. This last one will give the power to the leds and control it.
- A 5v pcb card ( that i found few weeks ago ) inside the box for an easy wiring to all the device and connectors.

It’s not only for one pi, i designed it to put any Raspberry.

Dimensions of the box : 15x15x5

the power will be outside to prevent all risk and to have something small.

i m glad to see you like it ☺️

**Comment by AmbiMod** [comment]
Hi my friend.

Some news.

i almost finished my setup. I designed a pcb because i need more connexion on it.

Normally i plan to sell it in a middle of august. I’m so exiting ☺️

BTW, did you see the new features on Hyperion NG ? Did you plan a release of hyperhdr soon ?

**Comment by awawa-dev** [comment]
Seems Hyperion NG team liked HyperHDR, at least some of its functionality. Too bad they didn't like MIT license. Still, they are aprox. a year behind.
New version of HyperHDR is almost finished but not plan to release soon as more testing is needed. Currently I'm building new LED strip setup with 96leds/meter sk6812 rgbw. And beside after new release the documentation would become outdated (tutorials on your blog also would become outdated and would need an update so no need to hurry ;) One of new feature that may interest you: CEC support is coming...not only to turn on/off device. And much more but it will be a surprise ;)
I can't wait for the final results of your project :)

**Reply by AmbiMod** [reply]
Ok nice to ear that the new version of hyperhdr will be available with new features ☺️

BTW awawa, my friend.  I want to give you a gift. A protoype version of my ambilight box. 

If you can send me your address, like that you can test it on your side ( mp on dealabs for exemple ).  if you want, it’s not an obligation. ( and i will understand if you don’t want ).

See you.

**Comment by AmbiMod** [comment]
Hi awawa.

I got some error message with my loop hdmi. Do you have an idea ?

![E0F1192A-558D-45DD-B664-EAE67EEFFEB9](https://user-images.githubusercontent.com/26428901/128638984-2eaebb56-cf2a-403b-89ba-c05a3234ee4d.png)

**Reply by awawa-dev** [reply]
Hi
Probably you have just restarted the grabber and then it happened for a short period of time.

**Reply by AmbiMod** [reply]
I have a power issue with this one. A lot of power is neccessary to make it working correctly.

BTW, i can’t ajust any capture resolution and frequency. I can just enable the quarter of frame mode to have a decent cpu usage, even on pi 4.

As you can see on this picture we have 960x540 so the half of fullhd.

![26DBB515-3067-49FB-85D8-508128014872](https://user-images.githubusercontent.com/26428901/128643599-c646b54b-f5d4-487b-be41-15b71d073f66.png)

I give you the cpu usage too

![5DE045E5-5492-4EE4-87D0-FC57F74611F6](https://user-images.githubusercontent.com/26428901/128643700-9a7d2387-45eb-4a3d-8bf9-b236a77865ef.png)

Do you think it’s possible to add another option doing the same but twice. And reduce btw the frequency to 30 ?

it’s a good loop but it need some tweaks 🙁

Ps: guys waiting the god of hdr on the Discord 😆

**Reply by awawa-dev** [reply]
I think I asked you to provide the full logs, because all the signs in the sky suggests it's MJPEG unfortunately ;) You can use software frame decimation set to 2 to have 30FPS. Errors after the grabber restart are typical and alone don't indicate power issues. 
I will join the discorq but must prepare "welcome cookies" first for your group :)

**Comment by awawa-dev** [comment]
Hi
Another small side project for next HyperHDR version  :)
https://github.com/awawa-dev/HyperSPI
